kratos/utilities/parallel_utilities.h

```diff
@@ -61,13 +61,22 @@ class BlockPartition
     BlockPartition(TIteratorType it_begin,
                    TIteratorType it_end,
                    int Nchunks = omp_get_max_threads())
-        : mNchunks(Nchunks)
     {
-        ptrdiff_t mBlockPartitionSize = (it_end-it_begin) / mNchunks;
+        KRATOS_ERROR_IF(Nchunks < 1) << "Number of chunks must be > 0 (and not " << Nchunks << ")" << std::endl;
```

Comment:
debug error?

Reply:
mmm, the good solution for this would be a c++20 concept :-D

Reply:
hm not sure abt the debug error. If it is wrong it is a division by zero later on.
Since it is only checked once at the beginning I would leave it as is

Reply:
ok. the comment was minor so it is already approved